scripts/backup.sh

```diff
@@ -15,3 +15,8 @@ if [ "$(id -u)" -eq 0 ]; then
 fi
 
 echo "backup created at $FILE_PATH"
+
+if [ "${DELETE_OLD_BACKUPS}" = true ]; then
+        echo "removing backups older than $OLD_BACKUP_DAYS days"
```

Comment:
there should be a check that checks if $OLD_BACKUP_DAYS has been set. Not sure what happens when the variable is empty

Reply:
Added a bash variable expansion to use 30 days as a default here